src/main/java/org/symphonyoss/integration/webhook/jira/parser/v2/CommentMetadataParser.java

```diff
@@ -182,6 +177,36 @@ private ObjectNode getCommentNode(JsonNode input) {
     return input.hasNonNull(COMMENT_PATH) ? (ObjectNode) input.path(COMMENT_PATH) : null;
   }
 
+  /**
+   * Jira's RTE syntax are not supported yet. For that reason, all jira formatting is removed in
+   * this method, and the user mentions are replaced by Nexus soft mention ([~user])
+   * @param comment The raw comment from Jira's webhook
+   * @return Comment supported by MessageML v2 syntax
+   */
+  private SafeString formatComment(String comment) {
+
+    if (StringUtils.isEmpty(comment)) {
+      return SafeString.EMPTY_SAFE_STRING;
+    }
+
+    comment = JiraParserUtils.stripJiraFormatting(comment);
+
+    SafeString safeComment = new SafeString(comment);
+
+    // FIXME: Uncomment me when soft mentions are supported on MessageML v2
+//    Map<String, User> usersToMention = determineUserMentions(comment);
+//    if (usersToMention != null && !usersToMention.isEmpty()) {
+//      for (Map.Entry<String, User> userToMention : usersToMention.entrySet()) {
+//        User user = userToMention.getValue();
+//
+//        safeComment.safeReplace(new SafeString(userToMention.getKey()),
+//            ParserUtils.presentationFormat(MENTION_MARKUP, user.getUsername()));
+//      }
+//    }
+    safeComment.replaceLineBreaks();
```

Comment:
You might use: SafeString safeComment = ParserUtils.escapeAndAddLineBreaks(comment);